source/common/protobuf/utility.cc

```diff
@@ -520,7 +521,7 @@ std::string MessageUtil::getYamlStringFromMessage(const Protobuf::Message& messa
   return out.c_str();
 }
 
-ProtobufUtil::StatusOr<std::string>
```

Comment:
also note that absl::StatusOr has move semantics even though ProtobufUtil::StatusOr doesn't (yet).

That might not help directly since ProtobufUtil::Status and absl::Status may be two different things, so that might not be a drop-in.